plugins/modules/remote_management/ism/is_add_user.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+# -*- coding:utf-8 -*-
+
+# Copyright(C) 2020 Inspur Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: is_add_user
+version_added: "0.1.0"
+author:
+    - WangBaoshan (@ISIB-group)
+short_description: Create user
+description:
+   - Create user on Inspur server.
+options:
+    uname:
+        description:
+            - User name of new user.
+        type: str
+        required: true
+    upass:
+        description:
+            - User password of new user.
+        type: str
+        required: true
+    role_id:
+        description:
+            - user role id of new user.
```

Comment:
```suggestion
            - User role id of new user.
```
Descriptions should always be proper sentences, i.e. start with an upper-case letter (if it is not an identifier that is correctly written lower-case) and end with a period.